tools/clang/test/CodeGenDXIL/hlsl/intrinsics/buffer-load-stores.hlsl

```diff
@@ -0,0 +1,213 @@
+// RUN: %dxc -DTYPE=float4    -DIX=SIx -T vs_6_6 %s | FileCheck %s
```

Comment:
This test sticks to non-aggregate types as LLVM defines them, which doesn't include vectors. This allows using typed buffers, which have a different code path from raw buffers.